tokio-trace/src/level_filters.rs

```diff
@@ -0,0 +1,121 @@
+use std::cmp::Ordering;
+use tokio_trace_core::Level;
+
+/// Describes the level of verbosity of a span or event.
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+pub struct LevelFilter(LevelFilterInner, Option<Level>);
+
+impl LevelFilter {
+    /// The "zero" level.
+    ///
+    /// Designates no logging.
+    pub const ZERO: LevelFilter = LevelFilter(LevelFilterInner::Zero, None);
```

Comment:
Minor naming nit, but: can we call this `OFF` instead of `ZERO`?
```suggestion
    pub const OFF: LevelFilter = LevelFilter(LevelFilterInner::Zero, None);
```